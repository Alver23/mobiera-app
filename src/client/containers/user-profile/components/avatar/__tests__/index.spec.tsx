// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Avatar } from 'react-native-elements';

// Under test file
import UserAvatar from '../index';

describe('<UserAvatar />', () => {
  let component: ShallowWrapper;
  const props = {
    image: 'http://localhost/img.png',
  };

  beforeEach(() => {
    component = shallow(<UserAvatar {...props} />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly', () => {
    expect(component.find(Avatar).exists()).toBeTruthy();
  });
});
