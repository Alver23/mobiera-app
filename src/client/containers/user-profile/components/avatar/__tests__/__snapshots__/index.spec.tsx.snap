// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserAvatar /> should save a snapshot of the component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "marginVertical": 12,
    }
  }
>
  <Themed.Avatar
    containerStyle={
      Object {
        "borderColor": "#ffffff",
        "borderWidth": 2,
      }
    }
    rounded={true}
    size="xlarge"
    source={
      Object {
        "uri": "http://localhost/img.png",
      }
    }
  />
</View>
`;
